pkg/ottl/e2e/e2e_test.go

```diff
@@ -353,6 +353,12 @@ func Test_e2e_converters(t *testing.T) {
 				tCtx.GetLogRecord().Attributes().PutStr("test", `<Log><Message>This is a log message!</Message><id>1</id></Log>`)
 			},
 		},
+		{
+			statement: `set(body, ConvertTextToElementsXML("<a><b/>foo</a>"))`,
```

Comment:
I forgot to ask before, could we also include another E2E test that uses the other parameters? Usually we like to test that all of the parameters parse, are in the right order, etc.

Reply:
Ah, wasn't aware of this standard but it makes perfect sense. I've added another test to exercise all parameters.

Reply:
No worries, we need to document that somewhere. Thanks for adding one.